tests/kafkatest/services/verifiable_producer.py

```diff
@@ -91,6 +92,10 @@ def _worker(self, idx, node):
                              self.compression_types[compression_index])
             producer_prop_file += "\ncompression.type=%s\n" % self.compression_types[compression_index]
 
+        if self.acks is not None:
+            self.logger.info("VerifiableProducer (index = %d) will use acks = %s", idx, self.acks)
+            producer_prop_file += "\nacks=%s\n" % self.acks
```

Comment:
Minor - `VerifiableProducer` actually has an `--acks` flag, so you could group this together with other flags in `start_cmd`